docs/reference/msbuild-targets.md

```diff
@@ -458,7 +461,7 @@ msbuild -t:build -restore
 
 The same logic applies to other targets similar to `build`.
 
-### Restoring PackageReference and packages.config with MSBuild
+### Restoring PackageReference and packages.config projects with MSBuild
```

Comment:
This breaks previous bookmarks, also update any anchors to:

#restoring-packagereference-and-packagesconfig-projects-with-msbuild

Reply:
Updated

Reply:
Updated bookmarks